src/wings_fr.lang

```diff
@@ -1300,12 +1300,12 @@
    ]},
   {info,
    [
-    {1,"Lumière ~s"},
-    {2,": Position ~s ~s ~s"}
+    {1,"Lumière ~ts"},
+    {2,": Position ~"}
```

Comment:
Single ~

Reply:
Thanks for the review. I fixed it using the online editor.